cloud/amazon/sns_topic.py

```diff
@@ -95,11 +94,15 @@ def canonicalize_endpoint(protocol, endpoint):
 
 
 
-def get_all_topics(connection):
+def get_all_topics(connection, module):
     next_token = None
     topics = []
     while True:
-        response = connection.get_all_topics(next_token)
+        try:
+            response = connection.get_all_topics(next_token)
+        except BotoServerError as e:
```

Comment:
I think @sivel's explanation in #1429 makes sense -- even if the module doesn't need to be Python 2.4 compatible, I agree with the desire to maintain consistent style throughout the codebase.